content/docs/pulumi-cloud/deployments/faq.md

```diff
@@ -21,6 +21,25 @@ aliases:
 
 Deployments run on single-use virtual machines and compute and storage are never shared across runs. We designed our architecture to maximize isolation. In addition, security features like OIDC allow you to fine tune credential scope, lifetime, and expiration policies at a per-deployment level. It is also possible to use [self-hosted runners](/docs/pulumi-cloud/deployments/customer-managed-agents/) if you require additional isolation.
 
+## Dependency Caching
+
+When using Pulumi-managed deployment agents, you have the option to speed up deployments using Dependency Caching.
+
+Caching method is simple: during the first deployment, deployment agent will automatically detect downloaded dependencies using lock files, zip them up and store the archive in blob storage. During all consequent deployments, agents will pull such an archive down and unpack it, saving time it would take to redownload those dependencies. When your dependencies change, deployment agents will automatically invalidate an old cache and create a new one.
```

Comment:
"The Caching method"
"the deployment agent"
I think you mean "subsequent" in "During all subsequent deployments"?
"saving time it would normally take..."

"invalidate the old cache" (there should only be one old cache)

Reply:
Articles kill me every time! The biggest flaw of the English language.
Thank you!